homeassistant/components/upnp/config_flow.py

```diff
@@ -273,4 +293,30 @@ async def _async_create_entry_from_discovery(
             CONFIG_ENTRY_MAC_ADDRESS: mac_address,
             CONFIG_ENTRY_HOST: discovery.ssdp_headers["_host"],
         }
-        return self.async_create_entry(title=title, data=data)
+        options = {
+            CONFIG_ENTRY_FORCE_POLL: False,
+        }
+        return self.async_create_entry(title=title, data=data, options=options)
+
+
+class UpnpOptionsFlowHandler(OptionsFlowWithConfigEntry):
```

Comment:
If possible, I'd split the options flow into a new PR and let this PR handle the registering of the specific keys to update

Reply:
Dropped the options flow. Is this what you meant, or did you mean to completely remove the `force_poll` related parts, such as https://github.com/home-assistant/core/pull/120262/files#diff-bf146bfe416cf0296a6c04f61fa6b8c1a9acb073dab44aab697d7f1eddc519d0R86 ?

Reply:
I think you do. Let me split it up a bit further.